src/Utils/Hooks/Mutations/useCreateSavedSearchMutation.ts

```diff
@@ -0,0 +1,59 @@
+import { Environment } from "react-relay"
+import { commitMutation, graphql } from "relay-runtime"
+import RelayModernEnvironment from "relay-runtime/lib/store/RelayModernEnvironment"
+import { useSystemContext } from "System/useSystemContext"
+import {
+  useCreateSavedSearchMutation,
+  CreateSavedSearchInput,
+  useCreateSavedSearchMutation$data,
+} from "__generated__/useCreateSavedSearchMutation.graphql"
+
+interface UseCreateSavedSearch {
+  relayEnvironment?: RelayModernEnvironment | Environment | undefined
+}
+
+export const useCreateSavedSearch = ({
+  relayEnvironment,
+}: UseCreateSavedSearch = {}) => {
+  const { relayEnvironment: defaultRelayEnvironment } = useSystemContext()
+
+  const createSavedSearch = (input: CreateSavedSearchInput) => {
+    return new Promise<useCreateSavedSearchMutation$data>((resolve, reject) => {
```

Comment:
We should use our modern form for creating mutations. See the settings app + hooks/useMutation hook

Reply:
Thanks, I'll take a look! 👍🏼